configuration/spring-config-server/src/main/resources/application.yml

```
@@ -1,15 +1,36 @@
-          username: ${GIT_USER}
-          password: ${GIT_TOKEN}
+        jdbc:
+          sql: "SELECT PROP_KEY, PROP_VALUE FROM PROPERTIES WHERE APPLICATION=? AND PROFILE=? AND LABEL=?"
+          default-label: main
+
+
+
+
+
+
 
```

YAMLlint (1.37.1)

[warning] too many blank lines (7 > 2)

(empty-lines)